internal/installation/platform.go

```diff
@@ -55,16 +55,29 @@ func matchPlatform(platforms []index.Platform, os, arch string) (index.Platform,
 	return index.Platform{}, false, nil
 }
 
-// osArch returns the OS/arch combination to be used on the current system. It
+// GoOSArch is wrapper around operating system and architecture
+type GoOSArch struct {
```

Comment:
Can we call this Platform or is that too generic?
We don’t care about “Go” part that much.

Reply:
It can be used in index too (since exported I didn't want to change in a lint PR).
They are also go specific so for now, it might leave as it is.

Reply:
:thinking: Wasn't `GoOSArch` that just added in this PR? I think we can choose whatever name suits best. Besides, everything is internal now, so we don't have to bother.

Reply:
Yes, I rename to whatever you want. 

What I mean is that with a better name it could be used in non-internal places too.

Reply:
Ok :)

So what about `Platform` as proposed by Ahmet?

Reply:
Sounds good to me since it's in the platform package. Renaming unless there is a better suggestion.

Reply:
ClientPlatform or OSArchPair isn't that bad either.

Reply:
I think renaming it to `Platform` could be misleading with `index.Platform`.

It's just OS and Arch so a simple naming might be better. `OSArchPair` sounds somehow better to me.